docs/Installing.asciidoc

```diff
@@ -288,15 +289,49 @@ OpenID method has its own `openid` section in `/etc/openqa/openqa.ini`:
 
 [source,ini]
 --------------------------------------------------------------------------------
+[auth]
+# method name is case sensitive!
+method = OpenID
+
 [openid]
 ## base url for openid provider
 provider = https://www.opensuse.org/openid/user/
 ## enforce redirect back to https
 httpsonly = 1
 --------------------------------------------------------------------------------
 
-OpenQA supports only OpenID version up to 2.0. Newer OpenID-Connect and OAuth is
-not supported currently.
+This method supports OpenID version up to 2.0.
+
+=== OAuth2
+
+Login via OAuth 2.0 is currently limited to GitHub.
+
+[source,ini]
+--------------------------------------------------------------------------------
+[auth]
+# method name is case sensitive!
+method = OAuth2
+
+[oauth2]
+provider = github
+key = mykey
+secret = mysecret
+--------------------------------------------------------------------------------
+
+In order to use GitHub for authorization, the instance needs to be registered on
+GitHub:
```

Comment:
I'd personally would prefer an inline link and not use so many blank lines, e.g.:

…the instance needs to be [registered on GitHub](https://github.com/settings/applications/new). Afterwards the key…